.github/workflows/pipeline-deploy-account-infrastructure.yaml

```diff
@@ -20,16 +20,29 @@ jobs:
         environment: dev
         domain: "uec-dos-service-management"
         workspace: ${{ needs.metadata.outputs.workspace }}
-        stacks: '["network", "aurora", "terraform_management", "github-runner"]'
+        stacks: '["network", "terraform_management", "github-runner"]'
```

Comment:
We'd still want to keep the "aurora" stack for the dev env just not for int & test

Reply:
Agreed aurora only in dev